Extension/ui/settings.html

```diff
@@ -581,10 +581,12 @@
         <div class="section-text" data-loc-id="c.standard.description">The version of the C language standard to use for IntelliSense. Note: GNU standards are only used to query the set compiler to get GNU defines, and IntelliSense will emulate the equivalent C standard version.</div>
         <div>
             <select name="inputValue" id="cStandard" class="select-default">
+                <option value="c2x">c2x</option>

```

Comment:
Should this be `c23`? Is there a reason this is `c2x` and not consistent with the other settings? Note: this file is the UI for the c_cpp_properties.json. The values in both the UI and .json for IntelliSense configuration should be the same.

Same comment for `gnu2x`

Reply:
thx, typo